platform/cosmos/model.go

```diff
@@ -0,0 +1,46 @@
+package cosmos
+
+// Tx - Base transaction object. Always returned as part of an array
+type Tx struct {
+	Block  string `json:"height"`
+	Date   string `json:"timestamp"`
+	ID     string `json:"txhash"`
+	TxData TxData `json:"tx"`
+}
+
+// TxData - "tx" sub object
+type TxData struct {
+	TxType     string     `json:"type"`
+	TxContents TxContents `json:"value"`
+}
+
+// TxContents - amount, fee, and memo
+type TxContents struct {
+	TxMessage []TxMessage `json:"msg"`
```

Comment:
No need to prefix everything with `Tx`, `Message`, `Fee`, `Memo` is fine

Reply:
Will remove, cheers